http_service/bugbug_http/models.py

```diff
@@ -26,6 +26,7 @@
 MODELS_NAMES = [
     "defectenhancementtask",
     "component",
+    "needsdiagnosis",
     "regression",
```

Comment:
I wasn't able to fully test this with the background worker as I got an error when adding `needsdiagnosis` to the list :(

`requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://community-tc.services.mozilla.com/api/index/v1/task/project.bugbug.train_needsdiagnosis.latest/artifacts/public/needsdiagnosismodel.zst`

Thinking this is because the artifact hasn't been created yet since there was the OOM error

Reply:
It is available now, the task with the latest version of bugbug (with your fix for the OOM included) ran 4 days ago.